libcodechecker/cmd/cmd_line_client.py

```diff
@@ -356,6 +357,50 @@ def get_report_data(client, reports, file_cache):
         return {'files': file_sources,
                 'reports': report_data}
 
+    def reports2report_data(reports):
```

Comment:
This function looks very familiar yet this patch only introduces it as block code. Can't we import the library where this is originally in (optionally with a proper handling of import errors?)

Reply:
This is a totally new code.  The function  get_report_data() a little bit above is similarish, but that is converting from another report format.